### PR TITLE
Code for review

### PR DESCRIPTION
**Pull request by abhaychauhan**
Commit handles following
- Design elements.
- Integration of elements with Emulator.

### CODE REVIEW

File: control/content/index.html [hunk 1]
```diff
@@ -1,65 +1,408 @@
...
-	<!-- JS -->
+<html ng-app='controllerapp'>
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <script src="../../../../scripts/buildfire.js"></script>
+
     <script src="../../../../scripts/angular/angular.min.js"></script>
+
+    <script src="../../../../scripts/angular/ui-bootstrap.min.js"></script>
+    <script src="../../../../scripts/jquery/jquery-1.11.2.min.js"></script>
```
Comment (DanielHindi) [inline control/content/index.html]: any reason why you are using both angular and jquery ?

File: control/content/index.html [hunk 2]
```diff
@@ -1,65 +1,408 @@
...
+
+    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
```
Comment (DanielHindi) [inline control/content/index.html]: i assume this section is bc you cant get it to work locally ?

File: plugin.json [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
+  }
+, "settings":{
+    "enabled":false
   }
+}
+,"widget":{
+
+},
+  "features" : [],
+  "languages" : ["en"]
```
Comment (DanielHindi) [inline plugin.json]: on the images in the resource solder. 1. why are these add 2.no dependencies should be in resources bc that folder wont be carried over to the app only the widget folder

Comment (abhaychauhan) [inline plugin.json]: I was unable to load the image at server at that time so for working of carousel i had to add that in resource folder .Now upon getting solution through email ,i am trying to fix it from image library.

File: control/content/index.html (hunk 1 above)
Comment (abhaychauhan) [inline control/content/index.html]: I required the angular.min.js file for working of angular tasks in our app.But that one preexist in our index.html file in comment section along with jquery-1.11.2.min.js .So by mistake that jquery-1.11.2.min.js  was  also included.I will remove that . Thanks for giving me feedback .But here the problem was working of carousel and modal pop up that was not working at that time.So i had to include external Bootstrap.js file

File: plugin.json (hunk 3 above)
Comment (DanielHindi) [inline plugin.json]: i found an issue in the plugin tester with imageLib we are fixing it

**Pull request merged**